### ISSUE TITLE
Pixelated Shadows

### ISSUE THREAD

**Issue by UrBoiTom** [X0nk/Bliss-Shader#366]
Could you add pixelated shadows to bliss? Here is an example of what it looks like in Rethinking voxels.
![image](https://github.com/user-attachments/assets/15d03940-97f3-446a-ad0c-b19e55ea5aa3)
![image](https://github.com/user-attachments/assets/8feac7af-5e24-4f50-9c6b-d0efbc1127bb)

**Comment by Sorro123** [comment]
I believe this line is where it is implemented https://github.com/gri573/rethinking-voxels/blob/1fd788f2c897169b7fc3375f0cb77526a2e65dc9/shaders/program/composite_lighting_fsh.glsl#L56

**Comment by X0nk** [comment]
i might. i have no opinion about it.

**Comment by Sorro123** [comment]
I feel like it would do well at hiding the weird stuff with the shadowmaps at high render distances

**Comment by X0nk** [comment]
weird stuff...?

**Comment by Sorro123** [comment]
Yeah, just jaggedness and flickering, but it could fix it, or at least make it less noticeable

**Comment by X0nk** [comment]
yeah it wont fix it, will just hide it.

**Comment by Sorro123** [comment]
Yeah but in this case hiding would fix the problem, at least for now or from the player's POV

**Comment by X0nk** [comment]
lol that's a little oxymoronic but i understand what you mean. what the user doesnt see doesnt exist
